### PR TITLE
feat(marketplace): allow specifying price as a number in listing creation

### PR DESCRIPTION
**Pull request by gregfromstl**

### CODE REVIEW

**Comment by gregfromstl** [comment]
* **#3441** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/3441?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gregfromstl and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/thirdweb/src/extensions/marketplace/direct-listings/write/createListing.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export type CreateListingParams = {
       /**
        * The price per token (in Ether)
        */
-      pricePerToken: string;
+      pricePerToken: string | number;
```
Comment (jnsdls) [inline packages/thirdweb/src/extensions/marketplace/direct-listings/write/createListing.ts:56]: if we want to do this we need to do it for everywhere we accept "ether" values, can't do it one-off just here

File: .changeset/loud-readers-repeat.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"thirdweb": minor
```
Comment (jnsdls) [inline .changeset/loud-readers-repeat.md:2]: this would be a patch if we did it (changing an existing api in a non-breaking way)

**Review by jnsdls — COMMENTED** [review]
lets discuss the need for this

**Pull request closed without merging**